modules/dr-testing-restore-procedures.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/about-disaster-recovery.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="dr-testing-restore-procedures_{context}"]
+= Testing restore procedures
+
+Testing the restore procedure is important to ensure that your automation and workload handle the new cluster state gracefully. Due to the complex nature of etcd quorum and the etcd Operator attempting to mend automatically, it is often difficult to correctly bring your cluster into a broken enough state that it can be restored. Follow this procedure to test such a broken state.
+
+[WARNING]
+====
+You **must** have SSH access to the cluster. Your cluster may be entirely lost without SSH access.
+====
+
+.Prerequisites
+
+* You have SSH access to control plane hosts.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Use SSH to connect to each of your nonrecovery nodes and run the following commands to disable etcd and the `kubelet` service:
+
+.. Disable etcd by running the following command:
++
+[source,terminal]
+----
+$ sudo /usr/local/bin/disable-etcd.sh
+----
+
+.. Delete variable data for etcd by running the following command:
++
+[source,terminal]
+----
+$ sudo rm -rf /var/lib/etcd
+----
+
+.. Disable the `kubelet` service by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl disable kubelet.service
+----
+
+. Exit every SSH session.
+
+. Run the following command to ensure that your nonrecovery nodes are in a `NOT READY` state:
++
+[source,terminal]
+----
+$ oc get nodes
+----
+
+. Follow the steps in "Restoring to a previous cluster state" in the "Additional resources" section to restore your cluster.
```

Comment:
```suggestion
. Follow the steps in "Restoring to a previous cluster state" to restore your cluster.
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations

> Do not state "in the Additional resources section" or other variants....

Reply:
I disagree strongly with this rule, but done.